README.md

```diff
@@ -331,6 +331,7 @@ For documentation on the latest development code see the [documentation index][d
 * [application_insights](./plugins/outputs/application_insights)
 * [aws kinesis](./plugins/outputs/kinesis)
 * [aws cloudwatch](./plugins/outputs/cloudwatch)
+* [azure_loganalytics](./plugins/outputs/azure_loganalytics)
```

Comment:
Should we rename to `azure_monitor_logs` now?

Based on note on https://docs.microsoft.com/en-us/azure/azure-monitor/platform/data-collector-api